taskcluster/ci/beetmover/kind.yml

```diff
@@ -17,14 +17,20 @@ kind-dependencies:
     - mac-notarization
     - repackage-signing
 
+task-defaults:
+    run-on-tasks-for: [action]
+    worker-type: beetmover
+    requires-level: 3
+    worker:
+        chain-of-trust: true
+        max-run-time: 1800
+    treeherder:
+        kind: build
+        tier: 1
+
+
 tasks:
     android-arm64:
-        requires-level: 3
```

Comment:
Are these changes/deletions related to the apt upload?

Reply:
Nope. removed that, :D